ramls/item.json

```diff
@@ -176,6 +176,34 @@
             "type": "boolean",
             "description": "Records the fact that the note should only be displayed for staff",
             "default": false
+          },
+          "source": {
+            "type": "object",
+            "description": "The user who added the note",
+            "properties": {
+              "id": {
+                "type": "string",
+                "description": "The id of the user who added the note"
+              },
+              "personal": {
```

Comment:
What is the purpose of storing the personal information of the user who created the note? Or are these derived properties, fetched on demand when the record is read?

Reply:
It is a requirement from the following story https://issues.folio.org/browse/MODINVSTOR-310.

The properties is fetched when the note is saved or updated. For example if the user created the note and another user updated, the information will change.

Reply:
As these properties are stored copies of information from the user (presumably to support sorting) they should be marked read only as this behaviour is controlled by the server (and this usage reflected in the description).

Reply:
Made source and date `readOnly` and updated the description.

Reply:
Thank you for updating this. Typically we mark all of the individual properties as read only, not only the object itself.

Reply:
Marked the indivudal properties as read only as well.